src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidator.cs

```diff
@@ -30,6 +30,13 @@
 namespace Microsoft.IdentityModel.Protocols.OpenIdConnect
 {
     /// <summary>
+    /// Delegate for validating additional claims in 'id_token' 
+    /// </summary>
+    /// <param name="idToken"><see cref="JwtSecurityToken"/> to validate</param>
+    /// <param name="context"><see cref="OpenIdConnectProtocolValidationContext"/> used for validation</param>
+    public delegate void IdTokenValidator(SecurityToken idToken, OpenIdConnectProtocolValidationContext context);
+
+    /// <summary>
```

Comment:
which validator you are referring to, the idTokenValidator? or the summary for the OpenIdConnectProtocolValidator.cs class?